api/v1alpha4/openstackcluster_webhook.go

```diff
@@ -17,12 +17,13 @@ limitations under the License.
 package v1alpha4
 
 import (
+	"fmt"
 	"reflect"
 
-	"github.com/pkg/errors"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/validation/field"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
```

Comment:
Not sure, but should v1alpha4 of CAPO use CAPI v1alpha4?
```suggestion
	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha4"
```

Reply:
For CAPO v0.5.0 we‘re using CAPI v1beta1 (see https://github.com/kubernetes-sigs/cluster-api-provider-openstack/issues/1024#issuecomment-968406388)